spec/rps_game_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rps_game'
+
+describe RPSGame do
+  subject(:game) { described_class.new(player_1_choice, player_2_choice) }
+  let(:player_1_choice) { 'Rock' }
+  let(:player_2_choice) { 'Scissors' }
+
```

Comment:
How come you're able to declare the subject first, even though it depends on player_1_choice and player_2_choice, which are declared on the following lines? :)